src/safe_apps/views.py

```diff
@@ -52,7 +65,13 @@ def get(self, request: Request, *args: Any, **kwargs: Any) -> Response:
         return super().get(request, *args, **kwargs)
 
     def get_queryset(self) -> QuerySet[SafeApp]:
-        queryset = SafeApp.objects.filter(visible=True)
+        only_listed = parse_boolean_query_param(
+            self.request.query_params.get("onlyListed", True)
```

Comment:
I think it might be preferable to have the default as `False`. An in when filtering a collection, if no filter is provided then I'd expect all the items of the collection.

It also makes querying any other Safe App easier. E.g.: If you want to query by an `url` on any app you'd have to set `onlyListed=false` to do that.

Reply:
Done in [501ce8f](https://github.com/safe-global/safe-config-service/pull/1111/commits/501ce8ffbbed691e5c1e3b48424860ba45489a29)